setup.py

```diff
@@ -46,8 +47,9 @@
     version=djangocms_versioning_filer.__version__,
     description=djangocms_versioning_filer.__doc__,
     dependency_links=[
-        "http://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms-4.0.0",
-        "http://github.com/divio/djangocms-versioning/tarball/master#egg=djangocms-versioning-0.0.24",
+        "https://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms-4.0.0",
+        "https://github.com/jonathan-s/djangocms-moderation/tarball/django2.0#egg=djangocms-moderation-1.0.22",
```

Comment:
Needs to be changed back once moderation is supporting django 2.2 for real. 
```suggestion
        "https://github.com/divio/djangocms-moderation/tarball/master#egg=djangocms-moderation-1.0.22",
```